components/BottomSheet/src/MDCBottomSheetController.h

```diff
@@ -110,6 +111,17 @@
  */
 @property(nonatomic, assign) MDCShadowElevation elevation;
 
+/**
+ This block is called after a change of the bottom sheet controller's elevation.
+
+  Use this block to respond to elevation changes in the alert controller or its ancestors.
+
+  @param bottomSheet This bottom sheet controller.
+ @param absoluteElevation The @c mdc_absoluteElevation this alert controller.
+ */
+@property(nonatomic, copy, nullable) void (^mdc_elevationDidChangeBlock)
+    (MDCBottomSheetController *_Nonnull bottomSheet, CGFloat absoluteElevation);
```

Comment:
Please change this block type to match the protocol, `id<MDCElevatable> _Nonnull bottomSheet`.

See #8105

Reply:
I `@synthesize`d to follow the pattern in #8106 .